root/components/tags.tt

```diff
@@ -4,7 +4,7 @@
 
 <h2>[% l("Tags") %]</h2>
 
-[%- IF tags.size OR c.user_exists -%]
```

Comment:
I probably added that because I figured you'd be able to edit tags on this page, but you can't, so.